contracts/optimistic-ethereum/libraries/codec/Lib_EIP155Tx.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+/* Library Imports */
+import { Lib_RLPReader } from "../rlp/Lib_RLPReader.sol";
+import { Lib_RLPWriter } from "../rlp/Lib_RLPWriter.sol";
+
+import { console } from "hardhat/console.sol";
+
+library Lib_EIP155Tx {
+    struct EIP155Tx {
+        uint256 nonce;
+        uint256 gasPrice;
+        uint256 gasLimit;
+        address to;
+        uint256 value;
+        bytes data;
+        uint8 recoveryParam;
+        uint8 v;
```

Comment:
`v` will overflow here I think

Reply:
I guess depending on what the chainid is

Reply:
Overflow doesn't matter for the `v` parameter. I'll add a comment that explains why.

Reply:
Added some more detailed comments.